### PR TITLE
Feature/connect

### PR DESCRIPTION
**Pull request by nwiranata**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+      <ion-toolbar>
+        <ion-title>
+          Assembly Church
+        </ion-title>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content>
+        <br/>
+        <ion-avatar margin>
+          <img src="https://cdn2.iconfinder.com/data/icons/avatar-profile/509/avatar_hair_contact_profile_user_default-512.png">
```
Comment (ltbrujahh) [inline src/app/app.component.html]: Can you try using this `src` instead? 

https://via.placeholder.com/512

File: src/app/pages/home/connect/connect.component.html [hunk 2]
```diff
@@ -1,34 +1,31 @@
...
+                        <ion-row>
+                            {{ group.name }}
+                            <br/>
+                            {{ group.description }}
+                        </ion-row>
                     </ion-col>
                     <ion-col>
-                        {{ group.description }}
+                        <br/>
+                        <ion-icon name="arrow-dropright"></ion-icon>
```
Comment (ltbrujahh) [inline src/app/pages/home/connect/connect.component.html]: Can this be a self-closing tag?

File: src/app/pages/home/connect/connect.component.scss [hunk 3]
```diff
@@ -4,10 +4,15 @@ ion-grid
     display: flex;
     flex-direction: column;
 
-    ion-row
-    {
-        flex: 1 0;
-    }
+    // ion-row
```
Comment (ltbrujahh) [inline src/app/pages/home/connect/connect.component.scss]: Probably just remove the comment, we can always just add it in later if we need it

File: src/app/pages/home/connect/connect.component.scss [hunk 4]
```diff
@@ -24,14 +29,12 @@ ion-grid
...
-.banner
-{
-    height: 20%;
-    background-color: lightblue;
-}
-
 .connect-groups
 {
-    height: 80%; // HACK: lets try to let this fill the rest of the page
+    height: 100%; // HACK: lets try to let this fill the rest of the page
```
Comment (ltbrujahh) [inline src/app/pages/home/connect/connect.component.scss]: I think this fixes the HACK? lets remove the comment in that case

File: src/app/pages/home/home.page.scss [hunk 5]
```diff
@@ -52,4 +52,8 @@ ion-grid
 .bg-connect
 {
     background-image: url('/assets/img/home/bg-connect.JPG');
+}
+
+ion-title {
+    font-size: 1em;
```
Comment (ltbrujahh) [inline src/app/pages/home/home.page.scss]: 😢 theres no new-line at EOF (end of file)

File: src/app/app.component.html (hunk 1 above)
Comment (nwiranata) [reply src/app/app.component.html]: i thought the vegeta avatar looked pretty cool  :(

Comment (ltbrujahh) [reply src/app/app.component.html]: Nevermind! I was too scared to open it at work 😆 ignore me!

File: src/app/pages/home/connect/connect.component.html (hunk 2 above)
Comment (nwiranata) [reply src/app/pages/home/connect/connect.component.html]: the br or ion-icon?

Comment (nwiranata) [reply src/app/pages/home/connect/connect.component.html]: br definitely

Comment (ltbrujahh) [reply src/app/pages/home/connect/connect.component.html]: the `ion-icon`, i think its a HTML standard where all tags can be self-closing

**Pull request merged**